feature/metis/conversation/src/main/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ui/reply/MarkdownTextField.kt

```diff
@@ -43,13 +57,25 @@ internal fun MarkdownTextField(
     topRightButton: @Composable RowScope.() -> Unit = {},
     onFocusAcquired: () -> Unit = {},
     onFocusLost: () -> Unit = {},
-    onTextChanged: (TextFieldValue) -> Unit
+    onTextChanged: (TextFieldValue) -> Unit,
+    onFileSelected: (Uri?) -> Unit = { _ -> },
 ) {
     val text = textFieldValue.text
+    val context = LocalContext.current
 
     var selectedType by remember { mutableStateOf(ViewType.TEXT) }
     var hadFocus by remember { mutableStateOf(false) }
 
+    val filePickerLauncher =
+        rememberLauncherForActivityResult(ActivityResultContracts.GetContent()) { uri ->
+            val mimeType = uri?.let { context.contentResolver.getType(it) }
```

Comment:
If uri is null here, do an early return. Then all the definitions of `onFileSelected` can be changed to `(Uri) -> Unit`.